builtin/aws/alb/README.md

```diff
@@ -1,3 +1,21 @@
-## Getting Started
+## AWS ALB
 
-TODO, Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+The AWS ALB plugin releases applications deployed to AWS by attaching [target
+groups](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-target-groups.html)
+to an [ALB](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html).
+
+### Components
+
+1. [ReleaseManager](./components/release-manager/README.md)
+
+### Related Plugins
+
+1. [AWS EC2](../ec2/README.md)
+2. [AWS Lambda](../lambda/README.md)
```

Comment:
@BrandonRomano This raises two alarms in my head:
1. I think we'll need to transform URLs to be _versioned/latest_
2. We might want to disallow relative paths (+ make this authoring experience easy)